apps/admin/backend/src/tabulation/full_results.test.ts

```diff
@@ -599,7 +654,7 @@ test('tabulateElectionResults - write-in handling', async () => {
           type: 'candidate',
           ballots: 189,
           overvotes: 30,
-          undervotes: 12,
```

Comment:
Another place where undervote count goes up because it is now accurate even when we don't fetch data from the write-in data.